lib/rules/property-no-vendor-prefix/__tests__/index.js

```diff
@@ -100,3 +100,46 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  config: [
+    true,
+    { ignoreProperties: ["transform", "columns", "/^animation-/"] }
+  ],
+
+  accept: [
+    {
+      code: "a { -webkit-transform: scale(1); }"
+    },
+    {
+      code: "a { -moz-columns: 2; }"
+    },
+    {
+      code: ".foo { -webkit-animation-delay: 0.5s; }"
+    },
+    {
+      code: ".foo { width: 320px; }"
+    }
+  ],
+  reject: [
+    {
+      code: "-webkit-border-radius: 10px;",
+      message: messages.rejected("-webkit-border-radius"),
+      line: 1,
+      column: 1
+    },
+    {
+      code: ".foo { -moz-background-size: cover; }",
+      message: messages.rejected("-moz-background-size"),
+      line: 1,
+      column: 8
+    },
+    {
+      code: ".foo { -WEBKIT-tranSFoRM: translateY(-50%); }",
```

Comment:
This shouldn't be rejected because `transform` is in ignored properties.

Reply:
See my comment above as why I believe this is valid.